src/main/java/core/mate/academy/model/BulldozerProducer.java

```diff
@@ -0,0 +1,31 @@
+package core.mate.academy.model;
+
+import core.mate.academy.service.MachineProducer;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Bulldozer> {
+
+    @Override
+    public List<Bulldozer> get() {
+        return createBulldozersList();
```

Comment:
you just call one method from anouther, if this method doesn't have any additional logic better not to have 2 methods

Reply:
done